services/Directory/data/FilterList.json

```diff
@@ -16804,5 +16804,46 @@
     "issuesUrl": "https://github.com/DandelionSprout/adfilt/issues",
     "licenseId": 35,
     "name": "Anti-'Gacha and Knockoff Games' List"
+  },
+  {
+    "description": "A blocklist of QAnon, conspiracy, fascist and fake news web sites.",
+    "homeUrl": "https://github.com/rimu/no-qanon",
+    "id": 2429,
+    "issuesUrl": "https://github.com/rimu/no-qanon/issues",
+    "name": "QAnon conspiracy nazi blocklist (IPv6)"
+  },
+  {
+    "description": "This is the list of bit notation bogons, aggregated, in text format. Updated as IANA allocations and special prefix reservations are made.",
+    "emailAddress": "sales@cymru.com",
+    "homeUrl": "https://team-cymru.com/community-services/bogon-reference/bogon-reference-http/",
+    "id": 2430,
+    "licenseId": 5,
+    "name": "Team Cymru Bogons"
+  },
+  {
+    "description": "The traditional bogon prefixes, plus prefixes that have been allocated to RIRs but not yet assigned by those RIRs to ISPs, end-users, etc.",
+    "emailAddress": "sales@cymru.com",
+    "homeUrl": "https://team-cymru.com/community-services/bogon-reference/bogon-reference-http/",
+    "id": 2431,
+    "licenseId": 5,
+    "name": "Team Cymru Fullbogons (IPv4)"
+  },
+  {
+    "description": "IPv6 “fullbogons”, all IPv6 prefixes that have not been allocated to RIRs and that have not been assigned by RIRs to ISPs, end-users, etc.",
```

Comment:
```suggestion
    "description": "IPv6 \"fullbogons\", all IPv6 prefixes that have not been allocated to RIRs and that have not been assigned by RIRs to ISPs, end-users, etc.",
```

Reply:
Oh, just saw your last commit hmm... one of these solutions should work. I hope. 🤷‍♂️

Reply:
I tried with `\“fullbogons\”` originally, which for whatever reason didn't work.